src/routes/+page.svelte

```diff
@@ -76,7 +76,8 @@
   >
     <h3>YouTube Feed</h3>
     <div class="my-3 text-gray-500">
-      A feed of Climate Town videos, right here in the Knowledge Hub!
+      A feed of Climate Town videos (plus some of our popular favorites), right
+      here in the Knowledge Hub!
```

Comment:
Going to change this to be a bit more explicit. Thanks for flagging it @JarrodBaniqued 
```suggestion
      A feed of Climate Town and various other climate YouTuber videos, right
      here in the Knowledge Hub!
```